egov/egov-collection/src/main/java/org/egov/collection/service/RemittanceServiceImpl.java

```diff
@@ -234,7 +234,7 @@
                             receipts = microserviceUtils.getReceipts(PaymentStatusEnum.NEW.name(),
                                     receipt.getService(),
                                     receipt.getFund(), receipt.getDepartment(), receipt.getReceiptDate());
-                            if (receipts != null) {
+                            if (!receipts.isEmpty() && receipts != null) {
```

Comment:
Is this make sense? what if receipts is null? First check not null then check not empty..